ui/learn/css/_screen.scss

```diff
@@ -18,6 +18,7 @@
   max-height: 100%;
   overflow: auto;
 
+  @media (prefers-reduced-motion: no-preference) {
```

Comment:
The body of this should be indented (or better, just run prettier and let it auto-format everything, either via the Prettier extension in your editor or by doing `yarn run format` at the root of the repo).